packages/amplify-category-api/src/amplify-graphql-transform/graphql-transformer-v2.ts

```diff
@@ -0,0 +1,48 @@
+import { AuthTransformer } from '@aws-amplify/graphql-auth-transformer';
+import { DefaultValueTransformer } from '@aws-amplify/graphql-default-value-transformer';
+import { FunctionTransformer } from '@aws-amplify/graphql-function-transformer';
+import { HttpTransformer } from '@aws-amplify/graphql-http-transformer';
+import { IndexTransformer, PrimaryKeyTransformer } from '@aws-amplify/graphql-index-transformer';
+import { MapsToTransformer } from '@aws-amplify/graphql-maps-to-transformer';
+import { ModelTransformer } from '@aws-amplify/graphql-model-transformer';
+import { PredictionsTransformer } from '@aws-amplify/graphql-predictions-transformer';
+import {
+  BelongsToTransformer,
+  HasManyTransformer,
+  HasOneTransformer,
+  ManyToManyTransformer,
+} from '@aws-amplify/graphql-relational-transformer';
+import { SearchableModelTransformer } from '@aws-amplify/graphql-searchable-transformer';
+import { TransformerPluginProvider } from '@aws-amplify/graphql-transformer-interfaces';
+import { TransformerFactoryArgs } from '../graphql-transformer/transformer-options-types';
+
+export const constructTransformerChain = (
+  customTransformers: TransformerPluginProvider[],
+  options?: TransformerFactoryArgs,
+): TransformerPluginProvider[] => {
+  const modelTransformer = new ModelTransformer();
```

Comment:
I see these transformers were instantiated this way in the old `getTransformerFactoryV2`, but do you know why these are not instantiated later like the others?

Reply:
Because some of the transformers are used as inputs to others (e.g index relies on model, manyToMany relies on hasMany, etc.)